crates/bevy_pbr/src/light/mod.rs

```diff
@@ -586,6 +579,16 @@ pub fn update_point_light_frusta(
             continue;
         }
 
+        let clip_from_view = Mat4::perspective_infinite_reverse_rh(
+            std::f32::consts::FRAC_PI_2,
+            1.0,
+            point_light.shadow_map_near_z,
+        );
+        let view_rotations = CUBE_MAP_FACES
+            .iter()
+            .map(|CubeMapFace { target, up }| Transform::IDENTITY.looking_at(*target, *up))
+            .collect::<Vec<_>>();
```

Comment:
Should this be moved in here? They don't change dependent on the point light?

Reply:
Reverted the change.